includes/apple-exporter/builders/class-component-layouts.php

```diff
@@ -92,7 +96,7 @@ public function set_anchor_layout_for( $component ) {
 			// The alignment position is the opposite of the body_orientation
 			// setting. In the case of centered body orientation, use left alignment.
 			// This behaviour was chosen by design.
-			if ( 'left' === $this->get_setting( 'body_orientation' ) ) {
+			if ( 'left' === $theme->get_value( 'body_orientation' ) ) {
```

Comment:
We didn't remove `get_setting`, but any settings that are actually theme settings need to use this new method. This is one of the reasons why I'm bumping the version to `1.3.0` because this could be a breaking change for some customizations.